x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/rest/action/enrollment/EnrollmentBaseRestHandler.java

```diff
@@ -30,7 +30,9 @@ protected Exception checkFeatureAvailable(RestRequest request) {
         if (failedFeature != null) {
             return failedFeature;
         } else if (XPackSettings.ENROLLMENT_ENABLED.get(settings) == false) {
-            return new ElasticsearchSecurityException("Enrollment mode is not enabled", RestStatus.FORBIDDEN);
+            return new ElasticsearchSecurityException("Enrollment mode [" + XPackSettings.ENROLLMENT_ENABLED.getKey() + "] is not " +
+                "configured",
```

Comment:
nit again: I find "not enabled" to be less confusing than "not configured" but I'll leave this up to you. ( It can "be configured" ==> explicitly set to false and we'd need to throw here.

Reply:
I think `"Enrollment mode [xpack.security.enrollment.enabled] is not enabled"` is a bit repetitive.

Reply:
May be `"Enrollment mode [xpack.security.enrollment.enabled] is not true"`? Or `not set`?

Reply:
"not set" has the same problem with "not configured" , it _can be set_, to *false* and we'd still throw. The confusion comes from having a single sentence that has 

- The "Enrollment mode" , which can be enabled or disabled
- The `xpack.security.enrollment.enabled` which can be set to `true` or `false`. 

We can either 

- revert to a more generic message like you had it originally "Enrollment mode is not enabled" or 
- expand a bit to  "Enrollment mode is not enabled. Set  [" + XPackSettings.ENROLLMENT_ENABLED.getKey() + "] to true, in order to use this API."

Reply:
I like the last one!